src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/ActivityTracking.cs

```diff
@@ -5,28 +5,68 @@
 using System.Collections.Generic;
 using System.Diagnostics.Tracing;
 using System.Linq;
+using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
 using Xunit;
+using System.Runtime.CompilerServices;
 
 namespace BasicEventSourceTests
 {
-    public class ActivityTracking
+    public class ActivityTracking : IDisposable
```

Comment:
This passed https://helixr1107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-113524-merge-23a8882c7437454e98/WasmTestOnChrome-ST-System.Diagnostics.Tracing.Tests/1/xharness-output/testResults.xml?helixlogtype=result